extension/chrome/elements/compose-modules/compose-send-btn-module.ts

```diff
@@ -224,7 +225,9 @@ export class ComposeSendBtnModule extends ViewModule<ComposeView> {
     await this.view.draftModule.draftDelete();
     this.isSendMessageInProgress = false;
     if (this.view.isReplyBox) {
-      this.view.renderModule.renderReplySuccess(msg, msgSentRes.id);
+      const listOfCc = ComposeRenderModule.getEmailHeaderFromComposeMessageBox('cc');
```

Comment:
There's absolutely no need in this method. All recipients - `to`, `cc` and `ccc` - are available in `msg.recipients`

Reply:
Thank you @limonte, the `msg.recipients` contains all the necessary headers to be checked.